src/graphql/subscriptions/patients.ts

```diff
@@ -0,0 +1,17 @@
+import gql from 'graphql-tag';
```

Comment:
should there be things in here for restoring and deleting patients? i noticed those were included in the backend PR but not here